concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1796,4 +1796,4 @@ Hale-1973-1798-1794	Miscellaneous-51	with		13b.31	1340	WITH
 Hale-1973-1798-1795	Miscellaneous-52	without		13b.32	1478	WITHOUT
 Hale-1973-1798-1796	Miscellaneous-53	of		13b.33		
 Hale-1973-1798-1797	Miscellaneous-54	except		13b.34		
-Hale-1973-1798-1798	Miscellaneous-55	like		13b.35		
+Hale-1973-1798-1798	Miscellaneous-55	like		13b.35	1117	LIKE
```

Comment:
not sure what the word here means, might as well be mapped to `1558 SIMILAR` https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L2166

Reply:
I agree. It should be mapped to SIMILAR.